test/srt/test_vision_openai_server_a.py

```diff
@@ -211,5 +211,36 @@ def setUpClass(cls):
         cls.base_url += "/v1"
 
 
+class TestPhi4mmServer(TestOpenAIVisionServer):
```

Comment:
Sorry I forgot to mention, there is actually an existing phi4mm test here: https://github.com/sgl-project/sglang/blob/7891bac16b0a905aacfbbe49709d740916555ae0/test/srt/test_vision_openai_server_b.py#L177